client/js/stores/day-store.js

```diff
@@ -48,7 +54,10 @@ var DayStore = assign({}, EventEmitter.prototype, {
 Dispatcher.register(function(payload){
   switch (payload.actionType){
     case AppConstants.ServerActionTypes.DAY_DATA_RECIEVED:
-      _dayData = payload.dayData;
+      _dayData.gigs = _.indexBy(payload.dayData, function(gig){
+        return gig.id;
+      });
+      _dayData.date = payload.dayData[0] && moment(payload.dayData[0].date, 'YYYY-MM-DD') || '';
```

Comment:
Looks like this is how you are trying to grab the current date. We are going to change the server responses to have the start_date, end_date on them so this won't be necessary soon. Can you comment this so we can remember to change it